src/inmanta/agent/forking_executor.py

```diff
@@ -768,65 +834,26 @@ def _make_child(self, name: str, log_level: int, cli_log: bool) -> tuple[multipr
         return p, parent_conn
 
     async def start(self) -> None:
-        self.running = True
-        self.cleanup_job = asyncio.create_task(self.cleanup_inactive_executors())
+        await super().start()
 
     async def stop(self) -> None:
-        self.running = False
+        await super().stop()
         await asyncio.gather(*(child.stop() for child in self.children))
 
     async def force_stop(self, grace_time: float) -> None:
+        await super().stop()
         await asyncio.gather(*(child.force_stop(grace_time) for child in self.children))
 
     async def join(self, thread_pool_finalizer: list[concurrent.futures.ThreadPoolExecutor], timeout: float) -> None:
+        await super().stop()
```

Comment:
This call was already made in the `stop()` method, so why do it here again?

Reply:
It was a precaution in case we would call join without calling stop afterward, but in the current implementation, we don't need this